director/commands/assets.py

```diff
@@ -49,13 +49,13 @@
         None,
     ),
     (
-        "https://cdn.jsdelivr.net/npm/vuetify@2.2.1/dist/vuetify.min.css",
-        "sha256-Ch5FMlNCdr6sBrwjHdGoTnAbAcKzE+UWiCXZHDT1HYk=",
+        "https://cdn.jsdelivr.net/npm/vuetify@2.3.8/dist/vuetify.min.js",
+        "sha256-b4RV6u+xflpPubfyN5gdooQRpDjROUcaCCSWbKQNX9Y=",
         None,
     ),
     (
```

Comment:
IMO, you have to add the `iconify.min.js` (v1.0.7) file link here (and related sha-256).